dnsapi/dns_acmedns.sh

```diff
@@ -1,31 +1,62 @@
 #!/usr/bin/env sh
 #
 #Author: Wolfgang Ebner
-#Report Bugs here: https://github.com/webner/acme.sh
+#Author: Sven Neubuaer
+#Report Bugs here: https://github.com/dampfklon/acme.sh
+#
+# Usage:
+# export ACMEDNS_BASE_URL="https://auth.acme-dns.io"
+#
+# You can optionally define an already existing account:
+#
+# export ACMEDNS_USERNAME="<username>"
+# export ACMEDNS_PASSWORD="<password>"
+# export ACMEDNS_SUBDOMAIN="<subdomain>"
 #
 ########  Public functions #####################
 
 #Usage: dns_acmedns_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+# Used to add txt record
 dns_acmedns_add() {
   fulldomain=$1
   txtvalue=$2
   _info "Using acme-dns"
-  _debug fulldomain "$fulldomain"
-  _debug txtvalue "$txtvalue"
+  _debug "fulldomain $fulldomain"
+  _debug "txtvalue $txtvalue"
+
+  ACMEDNS_BASE_URL="${ACMEDNS_BASE_URL:-$(_readaccountconf_mutable ACMEDNS_BASE_URL)}"
```

Comment:
Why is ACMEDNS_BASE_URL still saved in accountconf ?

Reply:
The idea is all acmesh Domains using acme-dns use the same acme-dns server. But each domain has its own account which it uses to publish the challenge.

Reply:
it looks not good to me.
I prefer that we save all the values in the same place.  no matter in domainconf or in accountconf.

Reply:
any update?

Reply:
From the intendet mode of operation the seperation makes sense for me. If you dont like the idea, I can duplicate the ACMEDNS_BASE_URL to all domain configurations.

Reply:
that will be great.
Thanks